launchers/default.sh

```diff
@@ -25,7 +25,8 @@ dt-exec roslaunch --wait \
     duckiebot_interface all_drivers.launch \
     veh:="$VEHICLE_NAME" \
     robot_type:="$ROBOT_TYPE" \
-    robot_configuration:="$ROBOT_CONFIGURATION"
+    robot_configuration:="$ROBOT_CONFIGURATION" \
+    dt_sensor_disable_camera:="$DT_SENSOR_CAMERA_DISABLE"
```

Comment:
this should go, the env var will be read by the launchfile directly

Reply:
Done in the new commit. Tested with lab DB21M.